modules/nw-about-multicast.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * networking/using-multicast.adoc
+
+[id="about-multicast-{context}"]
+= About multicast
+
+With IP multicast, data is broadcast to many IP addresses simultaneously.
+
+[IMPORTANT]
+====
+At this time, multicast is best used for low bandwidth coordination or service
+discovery and not a high-bandwidth solution.
+====
+
+Multicast traffic between {product-title} pods is disabled by default. If you
+are using the *OpenShiftSDN* network plug-in, you can enable multicast on a
+per-project basis.
+
+When using the *OpenShiftSDN* network plug-in:
+
+. Multicast packets sent by a pod will be delivered to all other pods in the
+project, regardless of `NetworkPolicy` objects. (Pods may be able to communicate
+over multicast even when they can't communicate over unicast.)
+. Multicast packets sent by a pod in one project will never be delivered to pods
+in any other project, even if there are `NetworkPolicy` objects allowing
+communication between the to projects.
```

Comment:
Should "to" be "two"? Or just remove the "to" to say "between the projects"?

Reply:
The latter makes the most sense, thanks!